src/nvim/if_cscope.c

```diff
@@ -1748,20 +1761,20 @@ static void cs_print_tags_priv(char **matches, char **cntxts, int num_matches)
  *
  * read a cscope prompt (basically, skip over the ">> ")
  */
-static int cs_read_prompt(int i)
+static int cs_read_prompt(size_t i)
 {
   int ch;
   char        *buf = NULL;   /* buffer for possible error message from cscope */
-  int bufpos = 0;
+  size_t bufpos = 0;
   char        *cs_emsg;
-  int maxlen;
+  size_t maxlen;
   static char *eprompt = "Press the RETURN key to continue:";
-  int epromptlen = (int)strlen(eprompt);
-  int n;
+  size_t epromptlen = strlen(eprompt);
 
   cs_emsg = _("E609: Cscope error: %s");
   /* compute maximum allowed len for Cscope error message */
-  maxlen = (int)(IOSIZE - strlen(cs_emsg));
+  assert(IOSIZE > strlen(cs_emsg));
```

Comment:
Yes, good catch. Since I'll make a change anyway I'll make sure it's stored in a variable.